README.md

```diff
@@ -18,11 +18,23 @@ the use of the code as-is in production applications.
 
 ## Documentation
 
-### API documentation
+### Customization of the update process
+
+Mender offers several ways to customize the update process. You can find a
+high-level overview of the different approaches in the documentation section about the
+[Customization of the update process](https://docs.mender.io/overview/customize-the-update-process).
+
+### Device-side API documentation
 
 The D-Bus APIs of the Mender client are documented in the [Device-side API
 section of the Mender Docs](https://docs.mender.io/device-side-api).
 
+### How to use the Mender Device-side API
+
+Please refer to the tutorial for
+[using the Device-side API](https://docs.mender.io/client-installation/use-the-device-side-api)
+for a step-by-step guide on how to use the example applications in this repository.
```

Comment:
Is this the new piece of docs you are writing? I can't see it in `development` site.

Also, is the tutorial there generic (so that the link is good in this README) or Update Flow Control centered (so that the link would be best in [the README for the specific example](https://github.com/mendersoftware/mender-client-python-example/tree/master/examples/update_flow_control))

Reply:
@lluiscampos yes, I'm writing it now, and this PR is not ready for review :)

Reply:
Sorry, I'll check this PR later then :+1: